jgt_tools/utils.py

```diff
@@ -21,9 +21,12 @@ def execute_command_list(commands_to_run, verbose=True):
     for command in commands_to_run:
         if verbose:
             print(f"+{command}")
-        job = subprocess.run(shlex.split(command), cwd=CONFIGS["base_dir"])
-        if job.returncode:
-            sys.exit(job.returncode)
+        try:
+            job = subprocess.run(shlex.split(command), cwd=CONFIGS["base_dir"])
+            if job.returncode:
+                sys.exit(job.returncode)
+        except KeyboardInterrupt:
+            pass
```

Comment:
This looks simple enough, I was going to approve it, but then I wondered why not do an exit?
My thinking is that we'd exit if the subprocess "failed", and for sure a Keyboard interrupt should be considered a failure? :-)

Reply:
My thought process was that if you have a chain of commands, say `a-b-c`, it could be `b` that has the `Ctrl+C` and then the follow up command `c` is some cleanup, so no need to necessarily exit out fully?

Reply:
I guess time will tell!